ietf/community/models.py

```diff
@@ -13,13 +12,23 @@
 from ietf.utils.models import ForeignKey
 
 class CommunityList(models.Model):
-    user = ForeignKey(User, blank=True, null=True)
+    person = ForeignKey(Person, blank=True, null=True)
     group = ForeignKey(Group, blank=True, null=True)
     added_docs = models.ManyToManyField(Document)
 
+    @property
```

Comment:
Why did you need this? We should purge the app of any knowledge of user. It really should only need to know about person.

Reply:
Might be useful as a stepping stone to make it raise `RuntimeError` and run the tests to see if it's being referenced.

Reply:
It's still in use in the templates, e.g. `{% url "ietf.community.views.feed" username=clist.user.username %}`. But I see below that you'd prefer to remove username from urls.

Reply:
Fixed the templates and a few other things that refer to user. I think the remaining references are related to usernames in requests or replies.